receiver/hostmetricsreceiver/internal/scraper/cpuscraper/cpu_scraper_linux.go

```diff
@@ -7,13 +7,19 @@
 package cpuscraper // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/cpuscraper"
 
 import (
+	"runtime"
+
 	"github.com/shirou/gopsutil/v3/cpu"
 	"go.opentelemetry.io/collector/pdata/pcommon"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/internal/metadata"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver/internal/scraper/cpuscraper/ucal"
 )
 
+func (s *scraper) recordCPUNumberDataPoint(now pcommon.Timestamp) {
+	s.mb.RecordSystemCPUNumberDataPoint(now, float64(runtime.NumCPU()))
```

Comment:
Maybe that is also not the right way to parse it here. But I assume its not to hard to get this value from somewhere else.

https://github.com/open-telemetry/opentelemetry-specification/pull/2292
https://github.com/open-telemetry/opentelemetry-specification/pull/2292

Reply:
Thanks for posting the links. Reading through https://github.com/open-telemetry/opentelemetry-specification/pull/2384, I'm more confident that the metric is a better option than a resource attribute. I'm good with accepting this PR. Still, I would like to hear back from @mx-psi and have another PR against the spec similar to https://github.com/open-telemetry/opentelemetry-specification/pull/2384 but from the collector perspective. The host metrics receiver can get an actual number of processors as opposed to the Java library.

Reply:
Funny, reading open-telemetry/opentelemetry-specification#2384 made me more convinced that this should be a resource attribute :smile: Let's continue the discussion over at open-telemetry/semantic-conventions/pull/99 and mark this as 'on hold' until that one is resolved.